src/lib/components/data-table/DataTableActions.svelte

```
@@ -65,69 +82,146 @@
-					<span class="sr-only">Open menu</span>
-					<EllipsisIcon />
-				</Button>
-			{/snippet}
-		</DropdownMenu.Trigger>
-		<DropdownMenu.Content>
-			{#if canEdit}
+	<div class="flex items-center gap-1">
+		<!-- INLINE ZONE -->
+		{#if hasInlineActions}
+			{#if canEdit && editInline}
 				<a href={editUrl}>
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)